tests/e2e/tests/e2e_happy_path/theia/DevWorkspaceHappyPath.spec.ts

```diff
@@ -57,7 +57,7 @@ const SpringAppLocators = {
 suite('Workspace creation via factory url', async () => {
     let factoryUrl : string = `${TestConstants.TS_SELENIUM_DEVWORKSPACE_URL}`;
 
-    const workspaceRootFolderName: string = 'src';
+    // const workspaceRootFolderName: string = 'src';
```

Comment:
What was the reason to comment the code?

Reply:
I forgot to remove the comment. It's been moved to line 34

Reply:
Thanks